sdp/processors/create_initial_manifest/create_initial_manifest_voxpopuli.py

```diff
@@ -136,7 +135,7 @@ def process_dataset_entry(self, data_entry: str):
         data = {
             "audio_filepath": tgt_wav_path,
             "duration": float(sox.file_info.duration(tgt_wav_path)),
-            "text": transcript_text,
+            "text": raw_text,
```

Comment:
Will this break any existing voxpopuli manifests?

Reply:
Maybe. I suppose it makes more sense for "text" to be the provided norm_text by default, so I will revert this change and work around it in the Spanish PC Voxpopuli config.